samples/02_power_users_developers/jupyter_dashboard_for_raster_analytics.ipynb

```diff
@@ -130,7 +130,7 @@
     "\n",
```

Comment:
<p>To make the search more robust, i think it's a good idea to include tags:<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%20on%20AWS%22" target="_blank">Landsat on AWS</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%22" target="_blank">Landsat</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multispectral%22" target="_blank">Multispectral</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multitemporal%22" target="_blank">Multitemporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22imagery%22" target="_blank">imagery</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22landsat%208%22" target="_blank">landsat 8</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22temporal%22" target="_blank">temporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22MS%22" target="_blank">MS</a> in the search. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/769/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>